From: Gregory Price <gourry@gourry.net>
Date: Thu, 26 Dec 2024 11:29:18 -0700
Subject: [PATCH] 
 mm-add-build-time-option-for-hotplug-memory-default-online-type-v4

update KConfig comments

Link: https://lkml.kernel.org/r/20241226182918.648799-1-gourry@gourry.net
Signed-off-by: Gregory Price <gourry@gourry.net>
Acked-by: David Hildenbrand <david@redhat.com>
Cc: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Cc: Huacai Chen <chenhuacai@kernel.org>
Cc: Jonathan Corbet <corbet@lwn.net>
Cc: Oscar Salvador <osalvador@suse.de>
Cc: "Rafael J. Wysocki" <rafael@kernel.org>
Cc: WANG Xuerui <kernel@xen0n.name>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 mm/Kconfig | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/mm/Kconfig b/mm/Kconfig
index af163dbbaab1..1b501db06417 100644
--- a/mm/Kconfig
+++ b/mm/Kconfig
@@ -554,7 +554,7 @@ choice
 	prompt "Memory Hotplug Default Online Type"
 	default MHP_DEFAULT_ONLINE_TYPE_OFFLINE
 	help
-	  Default memory type for driver managed hotplug memory.
+	  Default memory type for hotplugged memory.
 
 	  This option sets the default policy setting for memory hotplug
 	  onlining policy (/sys/devices/system/memory/auto_online_blocks) which
@@ -575,7 +575,7 @@ choice
 config MHP_DEFAULT_ONLINE_TYPE_OFFLINE
 	bool "offline"
 	help
-	  Driver managed memory will not be onlined by default.
+	  Hotplugged memory will not be onlined by default.
 	  Choose this for systems with drivers and user policy that
 	  handle onlining of hotplug memory policy.
 
@@ -583,22 +583,22 @@ config MHP_DEFAULT_ONLINE_TYPE_ONLINE_AUTO
 	bool "auto"
 	help
 	  Select this if you want the kernel to automatically online
-	  memory into the zone it thinks is reasonable. This memory
-	  may be utilized for kernel data (e.g. page tables).
+	  hotplugged memory into the zone it thinks is reasonable.
+	  This memory may be utilized for kernel data.
 
 config MHP_DEFAULT_ONLINE_TYPE_ONLINE_KERNEL
 	bool "kernel"
 	help
 	  Select this if you want the kernel to automatically online
-	  hotplug memory into a zone capable of being used for kernel
-	  data (e.g. page tables). This typically means ZONE_NORMAL.
+	  hotplugged memory into a zone capable of being used for kernel
+	  data. This typically means ZONE_NORMAL.
 
 config MHP_DEFAULT_ONLINE_TYPE_ONLINE_MOVABLE
 	bool "movable"
 	help
 	  Select this if you want the kernel to automatically online
 	  hotplug memory into ZONE_MOVABLE. This memory will generally
-	  not be utilized for kernel data (e.g. page tables).
+	  not be utilized for kernel data.
 
 	  This should only be used when the admin knows sufficient
 	  ZONE_NORMAL memory is available to describe hotplug memory,
-- 
2.39.5

